client-course-schedulizer/src/utilities/helpers/readCSV.ts

```diff
@@ -155,8 +155,10 @@ export const insertSectionCourse = (schedule: Schedule, section: Section, course
       section.instructionalMethod,
     );
 
-    schedule.courses[existingCourseIndex].name = course.name;
     schedule.courses[existingCourseIndex].department = course.department;
+    if (course.name !== schedule.courses[existingCourseIndex].name) {
```

Comment:
is this checking if a course name is already present in the list of course names?